### ISSUE TITLE
Ecological Society of America

### ISSUE THREAD

**Issue by mark02139** [CDL-Dryad/dryad-product-roadmap#2458]
**Name to appear on membership page:** 
Ecological Society of America

**Payment plan:**
Quarterly tiered
NOTE: I need a tiered invoicing report for ESA for Q1 (Jan-Mar 2023)

**Journals included:** 
Ecological Applications
Ecological Monographs
Ecology
Ecosphere
Frontiers in Ecology and the Environment

**Email contacts to receive submission notices: (note whether they receive all notices, or different lists for different journals)** 

**Email contacts to receive publication notices: (note whether they receive all notices, or different lists for different journals)** 

**ORCIDs of users who need access to admin dashboard: (they must log in to Dryad to set up their account before we can give them access, note whether each user administers a subset of journals)**

**Email contacts to discuss integration of journal metadata with Dryad:**

Development tasks:
- [x] add member name to membership page
- [x] update payment settings
- [x] update journal notification settings
- [x] add journal administrators
- [ ] start integration process (track on new ticket)
@ryscher @ahamelers @jleighherzog

**Comment by ryscher** [comment]
- Updated payment settings 
- Regenerated shopping cart report, including new tiered report
